### PR TITLE
Check if variable array key exists before execution method

### PR DESCRIPTION
**Pull request by Vlado-web**

### CODE REVIEW

**Comment by Vlado-web** [comment]
Fix 'Undefined array key "owl:maxCardinality"' bug.

**Comment by DjordyKoert** [comment]
This looks like an issue with an older version of api-platform https://github.com/api-platform/core/issues/5913. An update to https://github.com/api-platform/core/releases/tag/v3.2.2 should fix this.

**Comment by Vlado-web** [comment]
I am on the latest v3.2.10 and the problem persists.

File: OpenApiPhp/Util.php [hunk 1]
```diff
@@ -424,7 +424,7 @@ private static function mergeFromArray(OA\AbstractAnnotation $annotation, array
             if ('$ref' === $propertyName) {
                 $propertyName = 'ref';
             }
-            if (!\in_array($propertyName, $done, true)) {
+			if (($defaults[$propertyName] ?? null) && !\in_array($propertyName, $done, true)) {
```
Comment (DjordyKoert) [inline OpenApiPhp/Util.php:427]:
```suggestion
			if (!in_array($propertyName, $done, true) && array_key_exists($propertyName, $defaults)) {
```

**Review by DjordyKoert — CHANGES_REQUESTED** [review]

File: OpenApiPhp/Util.php (hunk 1 above)
Comment (Vlado-web) [reply OpenApiPhp/Util.php]: OK, applied, but I swapped conditions due to " This condition execution costs less than the previous one.".
I don't understand what a problem is with style? Formatting seems to be fine.

Comment (DjordyKoert) [reply OpenApiPhp/Util.php]: I think you are using tabs instead of spaces

Comment (Vlado-web) [reply OpenApiPhp/Util.php]: Exactly, thanks.

**Comment by DjordyKoert** [comment]
Thank you @Vlado-web 😄

**Comment by Vlado-web** [comment]
@DjordyKoert Thank you too, I like the bundle, your work is appreciated.

**Pull request merged**